packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/index.ts

```diff
@@ -11,28 +11,28 @@
  *     Obeo - initial API and implementation
  *******************************************************************************/
 
-export { DiagramRepresentation } from './representation/DiagramRepresentation';
-export { DiagramPaletteToolContext } from './renderer/palette/DiagramPaletteToolContext';
-export type { DiagramPaletteToolContextValue } from './renderer/palette/DiagramPalette.types';
-export { DiagramPaletteToolContribution } from './renderer/palette/DiagramPaletteToolContribution';
-export type { DiagramPaletteToolContributionComponentProps } from './renderer/palette/DiagramPaletteToolContribution.types';
-export type { Diagram, NodeData } from './renderer/DiagramRenderer.types';
-export type { GraphQLNodeStyleFragment } from './graphql/subscription/nodeFragment.types';
-export type { NodeTypeContextValue } from './contexts/NodeContext.types';
 export { NodeTypeContext } from './contexts/NodeContext';
-export type { ILayoutEngine, INodeLayoutHandler } from './renderer/layout/LayoutEngine.types';
-export type { DiagramNodeType } from './renderer/node/NodeTypes.types';
-export { GQLViewModifier } from './graphql/subscription/nodeFragment.types';
-export type { GQLNode, GQLNodeStyle } from './graphql/subscription/nodeFragment.types';
-export { BorderNodePositon } from './renderer/DiagramRenderer.types';
+export type { NodeTypeContextValue } from './contexts/NodeContext.types';
+export type { IConvertEngine, INodeConverterHandler } from './converter/ConvertEngine.types';
 export { convertLabelStyle } from './converter/convertDiagram';
 export { AlignmentMap } from './converter/convertDiagram.types';
-export type { IConvertEngine, INodeConverterHandler } from './converter/ConvertEngine.types';
-export { NodeTypeContribution } from './renderer/node/NodeTypeContribution';
-export { useDrop } from './renderer/drop/useDrop';
+export { GQLViewModifier } from './graphql/subscription/nodeFragment.types';
+export type { GQLNode, GQLNodeStyle, GraphQLNodeStyleFragment } from './graphql/subscription/nodeFragment.types';
+export { BorderNodePositon } from './renderer/DiagramRenderer.types';
+export type { Diagram, NodeData } from './renderer/DiagramRenderer.types';
+export { Label } from './renderer/Label';
 export { useConnector } from './renderer/connector/useConnector';
+export { useDrop } from './renderer/drop/useDrop';
 export { useDropNode } from './renderer/dropNode/useDropNode';
-export { Label } from './renderer/Label';
-export { DiagramElementPalette } from './renderer/palette/DiagramElementPalette';
-export * from './renderer/layout/layoutNode';
+export { ConnectionCreationHandles } from './renderer/handles/ConnectionCreationHandles';
```

Comment:
`export { ConnectionCreationHandles }` is the only actual change. It is needed for the custom EllipseNode.

Reply:
Thanks for mentioning this :)